server/constants.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
Could move to miniserver if this is used by only miniserver

Reply:
moved